ci-operator/config/redhat-developer/gitops-operator/redhat-developer-gitops-operator-master__v4.12.yaml

```diff
@@ -79,24 +90,35 @@ tests:
         requests:
           cpu: "2"
           memory: 6Gi
-    workflow: ipi-aws
+    workflow: generic-claim
 - as: kuttl-parallel
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    labels:
+      region: us-east-2
+    owner: openshift-ci
```

Comment:
This should either be `gitops` or `us-east-1`. There is no openshift-ci pool using us-east-2.